lib/stat_tracker.rb

```diff
@@ -188,6 +188,31 @@ def away_games_by_team
     end
   end
 
+  def home_or_away_games(where = "home")
+    @game_teams.select do |game|
+      game.hoa == where
+    end
```

Comment:
I like the consolidation of the home/away games - we will have to change my code to implement this as well